docs/freefeatures.rst

```diff
@@ -29,21 +29,13 @@ Simply include one command per line, typed exactly as you would inside a ``cmd2`
 Comments
 ========
 
-Comments are omitted from the argument list
-before it is passed to a ``do_`` method.  By
-default, both Python-style and C-style comments
-are recognized. Comments can be useful in :ref:`scripts`, but would
-be pointless within an interactive session.
+Any command line input where the first non-whitespace character is a # will be treated as a comment.
+Comments can be useful in :ref:`scripts`, but would be pointless within an interactive session.
```

Comment:
Maybe add that they can be useful in transcripts and put a reference to that documentation too?

Reply:
Transcript comments work differently and are documented at [transcript.html](https://cmd2.readthedocs.io/en/latest/transcript.html)

Reply:
Comments work normally within transcripts other than the special behavior at the beginning of a transcript.  So the one place where I could see these breaking changes potentially impacting ``cmd2`` users is if they used embedded comments to document their transcripts.  Not that I'm aware of anyone doing that, but I could see it being potentially useful.